### PR TITLE
Add Tierpoll WIP

### PR DESCRIPTION
**Pull request by ghost**
This is a work in progress feel free to help me out with this

TODO:

- [ ] Make it so it checks polls and return an error if there is a poll already going on

- [ ] Hardcode a poll

### CODE REVIEW

**Comment by TheRun98** [comment]
When all is done, make sure to run `npm test` after copying the bot git to C9, etc.

**Comment by ghost** [comment]
Ok

File: commands/features-tournaments.js [hunk 1]
```diff
@@ -10,6 +10,14 @@ function tryGetRoomName (room) {
 }
 
 exports.commands = {
+	tierpoll: 'nexttour',
+	tourpoll: 'nexttour',
+	nexttour: function (arg, by, room, cmd) {
+		//This can be set to whatever you want
+		this.reply("/tierpoll"); //Placeholder unless anyone wants to set the format
+		this.reply("/poll timer 1.5"); //minor bug where if you do /tierpoll while a poll is going on it sets the timer
```
Comment (DeathlyPlays) [inline commands/features-tournaments.js]: Is this needed?

**Review by DeathlyPlays — CHANGES_REQUESTED** [review]

File: commands/features-tournaments.js (hunk 1 above)
Comment (ghost) [reply commands/features-tournaments.js]: optional, i just put it there because i like a timer

Comment (DeathlyPlays) [reply commands/features-tournaments.js]: I just don't think it would be beneficial as 1.5 minutes, seeing as small servers may never get many votes

Comment (DeathlyPlays) [reply commands/features-tournaments.js]: If you made it like 3 or 5 that would be fine

Comment (ghost) [reply commands/features-tournaments.js]: Ok

**Comment by DeathlyPlays** [comment]
Why 8.0 @Gligar

**Comment by Mystifi** [comment]
Well, updating the tests to use the Node version we're trying to achieve is the first step to getting the tests to pass.

**Comment by ghost** [comment]
I should probably remake it..

**Pull request closed without merging**